tools/jsonschema2go/jsonschema2go/cli.go

```diff
@@ -75,7 +77,8 @@ func main() {
 		Package:              arguments["-o"].(string),
 		ExportTypes:          true,
 		URLs:                 parseStandardIn(),
-		DisableNestedStructs: true,
+		EnableDefaults:       arguments["-d"].(bool),
```

Comment:
Is this being used anywhere? Or is it passed somewhere downstream that accepts this property?